src/components/forms/VpnServerForm.js

```diff
@@ -0,0 +1,417 @@
+                ? `Select a VPC.`
+                : `Select at least one security group.`
+            }
+            className={classNameModalCheck}
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          {/* certificate_crn */}
+          <IcseTextInput
+            id={this.props.data.name + "-vpn-server-certificate-crn"}
+            field="certificate_crn"
+            componentName="certificate_crn"
+            tooltip={{
+              content:
+                "Secrets Manager certificate unique identifier for VPN server",
+              align: "top-left",
+            }}
+            labelText="Secrets Manager Certificate CRN"
+            value={
+              this.state.certificate_crn === undefined
+                ? ""
+                : String(this.state.certificate_crn)
+            }
+            onChange={this.handleInputChange}
+            invalid={invalidCRNs(this.state.certificate_crn).invalid}
+            invalidText={invalidCRNs(this.state.certificate_crn).invalidText}
+            className={classNameModalCheck}
+          />
+          {/* method toggle */}
+          <IcseSelect
+            formName={this.props.data.name + "-vpn-server-method"}
+            name="method"
+            labelText="Authentication Method"
+            groups={["Certificate", "Username"]}
+            value={titleCase(this.state.method)}
+            handleInputChange={this.handleInputChange}
+            className={classNameModalCheck}
+          />
+          {/* client_ca_crn */}
+          {this.state.method === "Certificate" && (
+            <IcseTextInput
+              id={this.props.data.name + "-vpn-server-client-ca-crn"}
+              field="client_ca_crn"
+              componentName="client_ca_crn"
+              labelText="Client Secrets Manager Certificate CRN"
+              value={
+                this.state.client_ca_crn === undefined
+                  ? ""
+                  : String(this.state.client_ca_crn)
+              }
+              onChange={this.handleInputChange}
+              invalid={invalidCRNs(this.state.client_ca_crn).invalid}
+              invalidText={invalidCRNs(this.state.client_ca_crn).invalidText}
+              className="fieldWidth"
+            />
+          )}
+          {/* client_ip_pool */}
+          <IcseTextInput
+            id={this.props.data.name + "-vpn-server-client-ip-pool"}
+            componentName="client_ip_pool"
+            name="client_ip_pool"
+            field="client_ip_pool"
+            value={this.state.client_ip_pool}
+            placeholder="x.x.x.x"
+            labelText="Client CIDR"
+            invalidCallback={() =>
+              this.props.invalidClientIpPoolCallback(this.state, this.props)
+            }
+            invalidText={this.props.invalidClientIpPoolTextCallback(
+              this.state,
+              this.props
+            )}
+            onChange={this.handleInputChange}
+            className="fieldWidthSmaller"
```

Comment:
should the classes following this have different sizing if in modal? modal looks a bit funky

Reply:
@isaaceichelberger Can you check how they look now? All inputs are meant to be "fieldWithSmaller" within the Modal except `Client Secrets Manager Certificate CRN` & `Client Idle Timeout (In Seconds)` as the labelText for these is too long so it skips a line and ruins the alignment.